webapp/channels/src/components/widgets/menu/menu_header.tsx

```diff
@@ -11,20 +10,19 @@ type Props = {
     onClick?: () => void;
 }
 
+const MenuHeader = ({children, onClick}: Props) => {
+    return (
+        <li
+            className='MenuHeader'
+            onClick={onClick}
+        >
+            {children}
+        </li>
+    );
+};
+
 /**
```

Comment:
This jsdoc comment should be above the component

Reply:
##Screenshots
before Component
![Screenshot 2023-10-10 172124](https://github.com/mattermost/mattermost/assets/121673023/e7db9f6a-88d8-4929-9b40-5d793ce1004b)

after component 
![Screenshot 2023-10-10 172152](https://github.com/mattermost/mattermost/assets/121673023/d8148ca2-c40f-4d05-abae-a28e808d7e72)

Before placing the deprecated notice,error is coming.
hey @M-ZubairAhmed ,please have a look through it.

Reply:
Hello @M-ZubairAhmed,
Could you please provide a solution for the issue at hand? Your assistance would be helpful.